bin/generate_truncated_json

```diff
@@ -114,7 +114,7 @@ def create_outlier_filename(in_file_name, window):
 def generate_truncated_json(json_file):
     """Generate an output file for each truncated data set."""
 
-    bin_dir = os.path.dirname(os.path.abspath(__file__))
+    warmup_stats_bin_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../warmup_stats/bin")
```

Comment:
In the other files we have avoided the use of `..` in filenames and instead used another call to `os.path.dirname`, we should probably be consistent here.

Reply:
Similarly `/` should probably be removed in favour of a call (or another argument) to `os.path.join`